libs/design/modal/src/modal-close/modal-close.directive.spec.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  Component,
+  DebugElement,
+} from '@angular/core';
+import {
+  waitForAsync,
+  ComponentFixture,
+  TestBed,
+} from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+
+import { DaffModalCloseDirective } from './modal-close.directive';
+import { DaffModalService } from '../service/modal.service';
+
+@Component({
+  template: `
+    <button daffModalClose></button>
+  `,
+})
+class WrapperComponent {}
+
+describe('@daffodil/design/modal | DaffModalCloseDirective', () => {
+  let wrapper: WrapperComponent;
+  let de: DebugElement;
+  let fixture: ComponentFixture<WrapperComponent>;
+
+  beforeEach(waitForAsync(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        DaffModalCloseDirective,
+        WrapperComponent,
+      ],
+      providers: [
+        DaffModalService,
+      ],
+    })
+      .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(WrapperComponent);
+    wrapper = fixture.componentInstance;
+    de = fixture.debugElement.query(By.css('[daffModalClose]'));
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```

Comment:
This is missing tests for the behavior of the id.

Reply:
added id test to `modal-title.directive.spec.ts`